prow/cmd/tide/main.go

```diff
@@ -38,6 +38,9 @@ import (
 	"k8s.io/test-infra/prow/metrics"
 	"k8s.io/test-infra/prow/pjutil"
 	"k8s.io/test-infra/prow/tide"
+
+	// Import storage providers
+	_ "k8s.io/test-infra/pkg/io/provider-imports"
```

Comment:
Would be nicer to have `pkg/io`/`io2` import `provider-imports` or the providers directly instead of forcing each cmd to do that

Reply:
You're absolutely right. I guess I had some circular dependencies in some version of this PR and then later on I didn't see that I could simplify it again.

Reply:
In the newest commits the import won't be necessary at all anymore.